### DISCUSSION TITLE
progress on new version

### DISCUSSION THREAD

**Discussion by The-Man-Is-Me** [3kh0/ext-remover#88] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Has anyone started work on a fix, or know if we even can. I know pingot is in construction, any progress report or another fix??

**Comment by ghost** [comment, marked as answer]
They are working on it right now, dev told me to put a keylogger on admin's computer

**Reply by The-Man-Is-Me** [reply]
Thanks

**Reply by Burst-lucky077** [reply]
yo hey bro is there a new one out yet

**Reply by ghost** [reply]
@Burst-lucky077 nope. check on when it was updated from time to time, though

**Reply by The-Man-Is-Me** [reply]
u get the keylogger on the admin yet :)

**Reply by ghost** [reply]
@The-Man-Is-Me unfortunately, I won't do that any time soon, as my usb drive is BROKEN. would you like me to upload the files?

**Reply by Huntlb** [reply]
> @The-Man-Is-Me unfortunately, I won't do that any time soon, as my usb drive is BROKEN. would you like me to upload the files?

Yes pls

**Reply by The-Man-Is-Me** [reply]
yes

**Reply by Alex2Mayhem** [reply]
> @The-Man-Is-Me unfortunately, I won't do that any time soon, as my usb drive is BROKEN. would you like me to upload the files?

Yes please

**Reply by md4r2** [reply]
@The-Man-Is-Me YUS :D

**Reply by Huntlb** [reply]
Wait how tf are you even supposed to sneak a keylogger onto the admin's computer wha

**Reply by The-Man-Is-Me** [reply]
here is his keylogger:
https://github.com/yeeteeyt/Dummykeylogger

**Reply by The-Man-Is-Me** [reply]
Also something interesting
https://github.com/MinhasKamal/TrojanCockroach

**Reply by Alex2Mayhem** [reply]
First of all who is the admin

**Reply by Alex2Mayhem** [reply]
And second how tf, am i gonna get a logger on like this?? Make it look like microsoft edge or somthing?

**Reply by ghost** [reply]
@Alex2Mayhem just run it, it's an undetectable trojan

**Reply by The-Man-Is-Me** [reply]
do you need to have the pc unlocked or will it just auto install??

**Reply by The-Man-Is-Me** [reply]
also who is admin??

**Reply by The-Man-Is-Me** [reply]
hey, @yeeteeyt  i was looking at your github and found your cavegame hack, but it does not load?? Could you fix it????

**Reply by ghost** [reply]
@The-Man-Is-Me I state in the repo that the site isn't working. Put the script in tampermonkey

**Reply by The-Man-Is-Me** [reply]
Thank you, sorry for bothering u guys so much, thanks for what you do.

**Reply by Burst-lucky077** [reply]
will this ever be back up

**Reply by ghost** [reply]
@Burst-lucky077 duh

**Reply by vawlk** [reply]
keylogger is a good way to get suspended or expelled.

**Reply by Theblackmamba824** [reply]
wuts keylogger

**Reply by The-Man-Is-Me** [reply]
a program that records every keystroke a user makes on a computer

**Reply by frogmasamune** [reply]
Who is the admin?
